### DISCUSSION TITLE
Is it possible to map the window size "reset"?

### DISCUSSION THREAD

**Discussion by chancez** [kovidgoyal/kitty#4685] [Q&A]
If you use cmd+r (start_resizing_window) there's an option for "R", "Reset", but I cannot find anyway to map this outside of the `start_resizing_window` mode like you can with `resize_window`. Is this possible? I tried `resize_window reset` and that doesn't work, it just widdens the window.

**Comment by chancez** [comment]
Self answer: `map cmd+shift+0 remote_control resize-window --axis reset`, though having this first class would be nice.

**Reply by kovidgoyal** [reply]
https://github.com/kovidgoyal/kitty/commit/1c48ec7196b042e284929eaae89125d6e523cc47
